src/main/java/de/tum/in/www1/artemis/service/programming/ProgrammingExerciseExportService.java

```diff
@@ -181,7 +181,14 @@ private void copyEmbeddedFiles(ProgrammingExercise exercise, Path outputDir, Lis
         Path embeddedFilesDir = outputDir.resolve("files");
         if (!embeddedFiles.isEmpty()) {
             if (!Files.exists(embeddedFilesDir)) {
-                Files.createDirectory(embeddedFilesDir);
+                try {
+                    Files.createDirectory(embeddedFilesDir);
+                }
+                catch (IOException e) {
+                    exportErrors.add("Could not create directory for embedded files: " + e.getMessage());
+                    log.warn("Could not create directory for embedded files. Won't include embedded files: " + e.getMessage());
```

Comment:
Should this be an error instead of a warning?

Reply:
Since this won't lead to an export failure, I wouldn't log it as an error but I'm not sure if we have specific guidelines, when we should which log level.

Reply:
A warning should be fine. We do this also for Exam exports if I'm not mistaken.